includes/functions_module.php

```diff
@@ -45,7 +45,7 @@ public function load($module, $default)
 
 		if (false === @include($this->modules_path . $this->modules_prefix . $module . '.' . $phpEx))
```

Comment:
Here is another silented include and I wonder why.
`@include($this->modules_path . $this->modules_prefix . $module . '.' . $phpEx)`